glean-core/rlb/src/lib.rs

```diff
@@ -42,6 +42,7 @@ use once_cell::sync::OnceCell;
 use std::collections::HashMap;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::Mutex;
+pub use whatsys;
```

Comment:
This shouldn't be necessary. As written this would re-export that crate, which we don't want to do.
Dependency crates are in the global namespace and thus already accessible without a `use` statement.